### DISCUSSION TITLE
Will this interface correctly show the user how to use wincuts, is there any suggustion on improvement?

### DISCUSSION THREAD

**Discussion by wy8933** [LyubomirT/wincuts#12] [Q&A] [answered]
Here is the updated UI:
![image](https://github.com/LyubomirT/wincuts/assets/112401719/218815d9-e628-47aa-892f-5a5b2331312c)

Here is the window that will pop up when the user clicks on the help button
![image](https://github.com/LyubomirT/wincuts/assets/112401719/db036456-2982-4132-814f-fae797739ee5)

**Comment by LyubomirT** [comment, marked as answer]
In my opinion, it's very well-structured, great job! If possible, I think it'd be great to explain Window Patch a little more extensively, and maybe add a scrollbar (because there will be more documentation in the future).
